EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -6399,7 +6399,7 @@ QUEST_LV_0300_20161005_006398	Investigate Inner Wall District 8 (8)	내성벽 
 QUEST_LV_0300_20161005_006399	Release the spell from the magic stone when the lamp magic stone gleams.	램프 마법석에 빛이 반짝이는 순간에 마법석의 마력을 해방하세요				
 QUEST_LV_0300_20161005_006400	Investigate Inner Wall District 8 (9)	내성벽 제 8구역 수사(9)				
 QUEST_LV_0300_20161005_006401	The Meaning of Inner Wall District 8	내성벽 제 8구역의 의미				
-QUEST_LV_0300_20161005_006402	Explaining what you have found while showing the clue memo	단서메모를 보여주며 조사해 온 것들에 대해 이야기하는 중				
+QUEST_LV_0300_20161005_006402	Explaining your findings	단서메모를 보여주며 조사해 온 것들에 대해 이야기하는 중				
```

Comment:
you removed the note about the memo?

Reply:
It's a 'progress bar' message (similar to picking up items that take a few seconds to collect)  that occurs when interacting with an NPC prior to the dialogue prompt. its former line was extremely cut off